app/Instructor/attendance/attendance.php

```diff
@@ -1 +1 @@
-<p>This is the attendance page. Put your content here.</p>
\ No newline at end of file
+<p>This is the attendance page. Put your content here.</p>
```

Comment:
@daxpatel1 I don't understand what you are trying to merge with this PR... Did you open the pull request with the wrong branch?